src/AutoPluginManager/i18n/es_ES.po

```diff
@@ -0,0 +1,228 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: AutoPluginManager\n"
+"POT-Creation-Date: 2024-11-30 02:03:46+0000\n"
+"PO-Revision-Date: 2024-11-30 13:55-0500\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: en_US\n"
```

Comment:
the same

Reply:
this one should be `es_ES` (and the file name should be `es-ES`)